src/io/config.cpp

```diff
@@ -22,26 +22,61 @@ void Config::KV2Map(std::unordered_map<std::string, std::string>* params, const
       value = Common::RemoveQuotationSymbol(Common::Trim(tmp_strs[1]));
     }
     if (key.size() > 0) {
-      auto value_search = params->find(key);
-      if (value_search == params->end()) {  // not set
-        params->emplace(key, value);
-      } else {
-        Log::Warning("%s is set=%s, %s=%s will be ignored. Current value: %s=%s",
-          key.c_str(), value_search->second.c_str(), key.c_str(), value.c_str(),
-          key.c_str(), value_search->second.c_str());
-      }
+      params->operator[](key).emplace_back(value);
     }
   } else {
     Log::Warning("Unknown parameter %s", kv);
   }
 }
 
+void GetFirstValue(const std::unordered_map<std::string, std::vector<std::string>>& params, std::string key, int* out) {
```

Comment:
maybe rename to "GetFirstIntValue" ? as it only supports int type?

Reply:
changed to `GetFirstValueAsInt` in e9a7a0f